.eslintrc

```diff
@@ -1,5 +1,8 @@
 {
```

Comment:
Can you add ESLint to the project explicitly (i.e. so `npm run lint` will run it?

Reply:
oh snap! 😊 I'll do it!